tests/safety/common.py

```diff
@@ -842,6 +842,12 @@ def test_tx_hook_on_wrong_safety_mode(self):
               continue
             if {attr, current_test}.issubset({'TestVolkswagenMqbSafety', 'TestVolkswagenMqbStockSafety', 'TestVolkswagenMqbLongSafety'}):
               continue
+            if {attr, current_test}.issubset({'TestChryslerRamDTSafety', 'TestChryslerRamDTLongitudinalSafety'}):
+              continue
+            # TODO: no sure this is OK
+            if {attr, current_test}.issubset({'TestChryslerSafety', 'TestChryslerLongitudinalSafety',
+                                              'TestChryslerRamHDSafety', 'TestChryslerRamHDLongitudinalSafety'}):
+              continue
```

Comment:
some addresses overlap between these platforms, is this the correct way to handle that?

Reply:
seems like this is correct since the overlapping addrs have the same definition